trimesh/io/wavefront.py

```diff
@@ -246,9 +265,16 @@ def append_mesh():
                     if len(f_split) > 1 and f_split[1] != '':
                         current['vt'].append(
                             attribs['vt'][int(f_split[1]) - 1])
+                    else:
+                        current['vt'].append([np.nan, np.nan, np.nan])
                     if len(f_split) > 2:
                         current['vn'].append(
                             attribs['vn'][int(f_split[2]) - 1])
+                    else:
+                        current['vn'].append([np.nan, np.nan, np.nan])
```

Comment:
`nan` is used for re-ordering afterwards.

Reply:
Adding NaN to the array makes me slightly nervous- is there any way we could do this with a boolean mask? Though I guess this way most of the lifting is done by the numpy constructor and is probably pretty fast.

Reply:
I'm not sure what is the best, but I updated in c71b473. Could you check that?

Reply:
Yeah that looks great, thanks!